lib/common/common/ByteSwap.h

```diff
@@ -50,6 +48,22 @@ RtlUlonglongByteSwap(
     _In_ ULONGLONG Source
     );
 #endif
-#endif
 
+#elif defined(__APPLE__)
```

Comment:
How is __APPLE__ get defined? ```https://github.com/dotnet/coreclr/blob/master/CMakeLists.txt``` deals with ```CLR_CMAKE_PLATFORM_DARWIN``` but not adding a symbol. Maybe we need to add one to ours?